lib/libspl/xdr.c

```diff
@@ -29,6 +29,7 @@
  * under license from the Regents of the University of California.
  */
 
+#ifndef WITH_TIRPC
```

Comment:
We should probably modify Makefile.in to omit this file entirely when libtirpc is used.